apps/angular/src/app/features/authentification/login/login.component.ts

```diff
@@ -0,0 +1,84 @@
+import { ChangeDetectionStrategy, Component, DestroyRef, inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { NonNullableFormBuilder, ReactiveFormsModule, Validators } from '@angular/forms';
+import { Login } from '@js-camp/core/models/login';
+import { MatCardModule } from '@angular/material/card';
+import { MatButtonModule } from '@angular/material/button';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { BehaviorSubject, catchError, finalize } from 'rxjs';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { UserService } from '@js-camp/angular/core/services/user.service';
+import { Router } from '@angular/router';
+import { URL_PATHS } from '@js-camp/core/utils/url-paths';
+import { FormErrorService } from '@js-camp/angular/core/services/form-error.service';
+import { ApiError } from '@js-camp/core/models/api-error';
+
+import { AuthComponent } from '../auth.component';
+import { InputPasswordComponent } from '../input-password/input-password.component';
+
+/** Login page component. */
+@Component({
+	selector: 'camp-login',
+	standalone: true,
+	templateUrl: './login.component.html',
+	styleUrl: './login.component.css',
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	imports: [
+		CommonModule,
+		ReactiveFormsModule,
+		MatCardModule,
+		MatButtonModule,
+		MatFormFieldModule,
+		MatInputModule,
+		AuthComponent,
+		InputPasswordComponent,
+	],
+})
+export class LoginComponent {
+	private readonly destroyRef = inject(DestroyRef);
+
+	private readonly fb = inject(NonNullableFormBuilder);
+
+	private readonly userService = inject(UserService);
+
+	private readonly router = inject(Router);
+
+	/** Form error service. */
+	protected readonly formErrorService = inject(FormErrorService);
+
+	/** Login form control group. */
+	protected readonly loginForm = this.fb.group({
+		email: ['', [Validators.required, Validators.email]],
+		password: ['', [Validators.required, Validators.minLength(8)]],
+	});
+
+	/** Loading state. */
+	protected readonly isLoading$ = new BehaviorSubject<boolean>(false);
+
+	/** Logs user with the provided credentials. */
+	public onSubmit(): void {
+		if (this.loginForm.invalid) {
+			return;
+		}
+		this.isLoading$.next(true);
+		const credentials = new Login(this.loginForm.getRawValue());
+		this.userService.login(credentials)
+			.pipe(
+				catchError((error: unknown) => {
+					if (error instanceof ApiError) {
+						this.formErrorService.showFormValidationErrors(this.loginForm, error);
+					}
+					throw Error;
+				}),
+				finalize(() => {
+					this.isLoading$.next(false);
+				}),
+				takeUntilDestroyed(this.destroyRef),
+			)
+			.subscribe(
+				_ => this.router.navigate([URL_PATHS.home]),
```

Comment:
If all arguments are unused, we prefer an empty arguments array.
```suggestion
				() => this.router.navigate([URL_PATHS.home]),
```